lib/crawler_web/templates/layout/app.html.heex

```diff
@@ -2,13 +2,17 @@
   <% info = get_flash(@conn, :info) %>
   <% error = get_flash(@conn, :error) %>
   <%= if info do %>
-    <div class="flex alert col-xl-6 alert-info justify-content-center m-auto">
-      <p class="my-0" role="alert"><%= info %></p>
+    <div class="flex alert alert-info justify-content-center m-auto">
+      <p class="my-0" role="alert">
+        <%= info %>
+      </p>
     </div>
   <% end %>
   <%= if error do %>
-    <div class="flex alert col-xl-6 alert-info justify-content-center m-auto">
-      <p class="my-0" role="alert"><%= error %></p>
+    <div class="flex alert alert-info justify-content-center m-auto">
```

Comment:
should we be `alert-danger` https://getbootstrap.com/docs/5.2/components/alerts/#examples